website/content/gateway/api_reference/resources/credits/credits.md

```diff
@@ -32,6 +32,12 @@ POST https://gateway.clearhaus.com/credits
 {{% regex_optional %}}Optional{{% /regex_optional %}}
 {{% /description_details %}}
 
+{{% description_term %}}purpose_of_payment {{% regex %}}[A-Z]{6}{{% /regex %}}{{% /description_term %}}
+{{% description_details %}} The fields code shall reflect the nature and purpose of the transaction.
+Valid codes are list in Visas Purpose of Payment - Standardized Codes List.
+{{% regex_optional %}} Required for Visa cross-border transactions destined for Argentina, Bangladesh, Chile, Columbia, Egypt, India and Mexico. {{% /regex_optional %}}
```

Comment:
```suggestion
{{% regex_optional %}}Required for Visa cross-border transactions destined for some specific countries; otherwise optional. See the partner guideline for details.{{% /regex_optional %}}
```

Maybe? 😬